assets/chat/js/formatters.js

```diff
@@ -198,6 +198,47 @@ class EmoteFormatter {
 
 }
 
+// ignore escaped backticks
+function findNextTick(str) {
+    var base = 0
+    while(str.length > 0){
+        var index = str.indexOf('\`') 
+        if (index == -1){
+            return -1
+        } else if (str.charAt(index-1) != '\\'){
+            return index+base
+        } else{
+            base += index+1
+            str = str.substring(index+1,)
+        }
+    }
+    return -1
+}
+
+function stringCodeParser(str) {
+    var indexOne = findNextTick(str)
+    if (indexOne != -1){
+        var afterTick = str.substring(indexOne+1,)
+        var indexTwo = findNextTick(afterTick)
+        if (indexTwo != -1){
+            var betweenTicks = afterTick.substring(0,indexTwo).replace(/\r?\n|\r/g, '')
```

Comment:
when would just `\r` happen?

Reply:
I used this as a guide https://stackoverflow.com/a/15433225/5698680

Reply:
I guess it's not _really_ needed.